packages/styled-components/rollup.config.js

```diff
@@ -123,7 +131,8 @@ const browserConfig = {
   plugins: configBase.plugins.concat(
     replace({
       __SERVER__: JSON.stringify(false),
-    })
+    }),
+    minifierPlugin
```

Comment:
If we're minifying all output bundles (which I'm a fan of) I'd love to make sure to also try out Closure Compiler, which is pretty amazing https://github.com/styled-components/styled-components/blob/d220a202ffc590447d32ce3b5b79e61fe939308b/packages/styled-components/rollup.config.js#L111-L137

This is what we're currently doing for `urql` at least, where we try to squeeze out a lot of bytes here and there https://github.com/FormidableLabs/urql/blob/e031f9b887960a8113890777e8a100973068a36c/scripts/rollup/plugins.js#L98-L137

Reply:
Shaved off 0.4kB!

Reply:
So although it reduced the bundle size, something that terser doing actually regressed perf here. Going to hold off on it for now.